src/index.js

```diff
@@ -1,91 +1,24 @@
-const fs = require("fs"),
-    http = require("http"),
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler");
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+import morgan from 'morgan';
+import routes from './routes';
 
-const isProduction = process.env.NODE_ENV === "production";
-
-// Create global app object
 const app = express();
+const port = process.env.PORT || 3000;
 
 app.use(cors());
-
-// Normal express config defaults
-app.use(require("morgan")("dev"));
+app.use(morgan('dev'));
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
```

Comment:
This could actually be `app.use(express.json)` so we can do away with the redundant body-parser